parsers/CL.py

```diff
@@ -91,11 +134,25 @@ def fetch_production(zone_key='CL', session=None, target_datetime=None, logger=l
     """
 
     if target_datetime is None:
-        target_datetime=arrow.now(tz='Chile/Continental')
-        logger.warning('The real-time data collected by the parser is incomplete for the latest datapoints/hours,'
-                       'so the last 3 datapoints were omitted.'
-                       'If desired, please specify a historical date in YYYYMMDD format.')
+        gd = get_data_live(session, logger)
+        
+        generation = production_processor_live(gd, zone_key)
+
+        datapoint = {
+                      'zoneKey': zone_key,
+                      'datetime': generation[0].datetime,
+                      'production': generation[1],
+                      'storage': {
+                          'hydro': None,
+                                  },
+                      'source': 'panelapp.coordinadorelectrico.cl'
```

Comment:
```suggestion
                      'source': 'coordinadorelectrico.cl'
```
We don't generally bother with the sub domain parts.